src/components/FeaturePanel/QuickActions/ShareDialog/ShareDialog.tsx

```diff
@@ -43,6 +44,7 @@ export const ShareDialog = ({ open, onClose }: Props) => {
       </DialogTitle>
       <DialogContent sx={isMobileMode ? undefined : { minWidth: 480 }}>
         <LinkSection />
+        <QrCodeSection />
```

Comment:
Nice - this is the proper way to do it. No props, and using the context inside whenever possible 👍